packages/model-typings/src/models/IUsersModel.ts

```diff
@@ -379,4 +379,9 @@ export interface IUsersModel extends IBaseModel<IUser> {
 	countRemote(options?: FindOptions<IUser>): Promise<number>;
 	findOneByImportId(importId: string, options?: FindOptions<IUser>): Promise<IUser | null>;
 	removeAgent(_id: string): Promise<UpdateResult>;
+	findAgentsWithDepartments<T = ILivechatAgent>(
+		roles: string,
```

Comment:
Isn't this an array of strings also? 🤔

Reply:
No it isn't. If you check the typing here it is a string -> https://github.com/RocketChat/Rocket.Chat/blob/b198164368ac6d56101869c63ac073b3b8ee8b3a/apps/meteor/app/livechat/server/api/lib/users.ts#L15

But I'm glad you raised this because the variable name definitely needs to be updated here to reflect it is singular and not plural